### PR TITLE
LPD-41872 Remove some of the duplicate entries for ListTypeDefinition and ListTypeEntry

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-41872

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json [hunk 1]
```diff
@@ -16,817 +16,514 @@
...
 					"externalReferenceCode": "url",
 					"key": "url",
-					"name": "URL",
-					"name_i18n": {
-						"en-US": "URL"
-					}
+					"name": "URL"
 				}
 			],
-			"name": "jobDefinitionParameterType",
```
Comment (marcelabc) [inline workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json:148]: You changed it here. Was it intentional?

File: workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json [hunk 2]
```diff
@@ -16,817 +16,514 @@
...
-			"name_i18n": {
-				"en-US": "Job Definition Parameter Type"
-			}
+			"name": "Job Definition Parameter Type"
```
Comment (marcelabc) [inline workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json:100]: You changed it here. Was it intentional?

File: workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json [hunk 3]
```diff
@@ -16,817 +16,514 @@
...
-						"en-US": "Upstream"
-					}
+					"name": "Upstream"
 				}
 			],
-			"name": "gitBranchType",
-			"name_i18n": {
-				"en-US": "Git Branch Type"
-			}
+			"name": "Git Branch Type"
```
Comment (marcelabc) [inline workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json:27]: You changed it here. Was it intentional?

File: workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json [hunk 4]
```diff
@@ -16,817 +16,514 @@
...
-						"en-US": "Slave"
-					}
+					"name": "Slave"
 				}
 			],
-			"name": "jenkinsNodeType",
-			"name_i18n": {
-				"en-US": "Jenkins Node Type"
-			}
+			"name": "Jenkins Node Type"
```
Comment (marcelabc) [inline workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json:43]: You changed it here. Was it intentional?

File: workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json [hunk 5]
```diff
@@ -16,817 +16,514 @@
...
-						"en-US": "Upstream Git Branch Name"
-					}
+					"name": "Upstream Git Branch Name"
 				}
 			],
-			"name": "jobComparatorType",
-			"name_i18n": {
-				"en-US": "Job Comparator Type"
-			}
+			"name": "Job Comparator Type"
```
Comment (marcelabc) [inline workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json:84]: You changed it here. Was it intentional?

File: workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json [hunk 6]
```diff
@@ -16,817 +16,514 @@
...
 					"externalReferenceCode": "verificationServersDaily",
 					"key": "verificationServersDaily",
-					"name": "Verification Servers Daily",
-					"name_i18n": {
-						"en-US": "Verification Servers Daily"
-					}
+					"name": "Verification Servers Daily"
 				}
 			],
-			"name": "jobType",
```
Comment (marcelabc) [inline workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json:573]: You changed it here. Was it intentional?

File: workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json [hunk 7]
```diff
@@ -16,817 +16,514 @@
...
 					"externalReferenceCode": "portalSource",
 					"key": "portalSource",
-					"name": "Portal Source",
-					"name_i18n": {
-						"en-US": "Portal Source"
-					}
+					"name": "Portal Source"
 				}
 			],
-			"name": "liferayBundle",
```
Comment (marcelabc) [inline workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json:630]: You changed it here. Was it intentional?

File: workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json [hunk 8]
```diff
@@ -16,817 +16,514 @@
...
+					"name": "7.3.x"
 				},
 				{
 					"externalReferenceCode": "ee61x",
 					"key": "ee61x",
-					"name": "ee61x",
-					"name_i18n": {
-						"en-US": "ee-6.1.x"
-					}
+					"name": "ee-6.1.x"
```
Comment (marcelabc) [inline workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-batch/batch/00-list-type-definition.batch-engine-data.json:430]: You changed it here. Was it intentional?

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:report:65967423

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**